bokeh/_templates/autoload_js.js

```diff
@@ -76,7 +83,7 @@ calls it with the rendered model.
     console.log("Bokeh: BokehJS loaded, going straight to plotting");
```

Comment:
if we changed to `if (typeof(window._bokeh_is_loading) !== undefined && window._bokeh_is_loading == 0` would that be more robust? Or else `if (typeof(Bokeh) !== undefined && typeof(Bokeh.SomeWidget) !== undefined)` maybe?